lib/davinci_dtr_test_kit/dtr_light_ehr_suite.rb

```diff
@@ -1,18 +1,47 @@
+require 'tls_test_kit'
 require_relative 'version'
+require_relative 'dtr_options'
+require 'smart_app_launch/smart_stu1_suite'
+require 'smart_app_launch/smart_stu2_suite'
 
 module DaVinciDTRTestKit
   class DTRLightEHRSuite < Inferno::TestSuite
     id :dtr_light_ehr
     title 'Da Vinci DTR Light EHR Test Suite'
-    description %(
-        # Da Vinci DTR Light EHR Test Suite
+    description <<~DESCRIPTION
+      The Da Vinci DTR Test Kit Light EHR Suite validates the conformance of SMART apps
+      to the STU 2 version of the HL7® FHIR®
+      [Da Vinci Documentation Templates and Rules (DTR) Implementation Guide](https://hl7.org/fhir/us/davinci-dtr/STU2/).
 
-        This suite validates that an EMR or other application
-        can act as a data source for a DTR SMART App. Inferno
-        will act as a DTR SMART App making requests for data
-        against the system under test and storing completed
-        questionnaire responses.
-      )
+      ## Scope
+
+      These tests are a **DRAFT** intended to allow app implementers to perform
+      preliminary checks of their systems against DTR IG requirements and [provide
+      feedback](https://github.com/inferno-framework/davinci-dtr-test-kit/issues)
+      on the tests. Future versions of these tests may validate other requirements and may change the test validation logic.
+
+      ## SMART App Launch
+
+      Use this information when registering Inferno as a SMART App:
+
+      * Launch URI: `#{SMARTAppLaunch::AppLaunchTest.config.options[:launch_uri]}`
+      * Redirect URI: `#{SMARTAppLaunch::AppRedirectTest.config.options[:redirect_uri]}`
+
+      If a client receives a SMART App Launch card in a response and would like
+      to test their ability to launch Inferno as a SMART App, first run the
+      SMART on FHIR Discovery and SMART EHR Launch groups under FHIR API >
+      Authorization. When running the SMART EHR Launch group, Inferno will wait
+      for the incoming SMART App Launch request, and this is the time to perform
+      the launch from the client being tested.
+
+      ## Running the Tests
```

Comment:
add details in this section, particularly regarding how to use the reference implementation preset such as the relevant information about how to get inferno-qa reference server to do an EHR launch (I think it is here: To launch an app from the EHR go to `reference-server/app/app-launch`).

Reply:
would still like to see some basic instructions here.

Reply:
oops - I see them now in the file, but the file isn't getting loaded from the light ehr suite yet. Fix that an it should be good!